### PR TITLE
doc: add description about estargz and nerdctl support

### PR DESCRIPTION
**Pull request by imeoer**
Signed-off-by: Yan Song <yansong.ys@antfin.com>

### CODE REVIEW

**Comment by yqleng1987** [comment]
@imeoer , a new test job has been submitted. Please wait in patience.

File: README.md [hunk 1]
```diff
@@ -126,6 +128,10 @@ Browse the documentation to learn more. Here are some topics you may be interest
 
 - Nydus can also run with macfuse(a.k.a osxfuse).For more details please read [nydus with macos](./docs/nydus_with_macos.md).
 
+## Run eStargz image
```
Comment (hsiangkao) [inline README.md:131]: Run lazy-pulling-enabled eStargz images with nydus-snapshotter

**Comment by yqleng1987** [comment]
@imeoer , your pull request has been updated, but a test job is already running. A new test job wil be submitted after the current job is finished.

**Comment by yqleng1987** [comment]
@imeoer , your test job is running now. Please wait in patience.

**Comment by yqleng1987** [comment]
@imeoer , your pull request has been updated, but a test job is already running. A new test job wil be submitted after the current job is finished.

**Comment by yqleng1987** [comment]
@imeoer , your test job is running now. Please wait in patience.

**Comment by yqleng1987** [comment]
@imeoer , The CI test is completed, please check result:
<table>
    <tr>
        <th colspan="2">Test Case</th><th>Test Result</th>
    </tr>
    <tr>
        <td colspan="2">merge-target-branch</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6174>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">build-docker-image</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6174>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">compile-nydus</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6174>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">compile-ctr-remote</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6174>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">compile-nydus-snapshotter</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6174>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">start-nydus-snapshotter-config-containerd</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6174>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">run-container-with-nydus-image</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6174>:white_check_mark:SUCCESS</a></td>
    </tr>
</table>

Sorry, the current test job result is not valid, because your pull request has been updated, and a new test job will be submitted.

File: README.md [hunk 2]
```diff
@@ -126,6 +128,10 @@ Browse the documentation to learn more. Here are some topics you may be interest
...
+
+Use the containerd remote snapshotter plugin [nydus-snapshotter](https://github.com/containerd/nydus-snapshotter), which supports both nydus image runs and direct support for [eStargz](https://github.com/containerd/stargz-snapshotter) image runs with appending `--enable-stargz` command line option.
```
Comment (hsiangkao) [inline README.md:133]: which supports running both Nydus images and [eStargz](https://github.com/containerd/stargz-snapshotter) images with `--enable-stargz` command line option appended.

**Comment by yqleng1987** [comment]
@imeoer , your pull request has been updated. A new test job will be submitted. Please wait in patience.

**Comment by yqleng1987** [comment]
@imeoer , the test job has been submitted. Please wait in patience.

**Comment by yqleng1987** [comment]
@imeoer , your pull request has been updated, but a test job is already running. A new test job wil be submitted after the current job is finished.

**Comment by yqleng1987** [comment]
@imeoer , your test job is running now. Please wait in patience.

**Review by hsiangkao — APPROVED** [review]

**Comment by yqleng1987** [comment]
@imeoer , your pull request has been updated, but a test job is already running. A new test job wil be submitted after the current job is finished.

**Comment by yqleng1987** [comment]
@imeoer , your test job is running now. Please wait in patience.

**Comment by yqleng1987** [comment]
@imeoer , The CI test is completed, please check result:
<table>
    <tr>
        <th colspan="2">Test Case</th><th>Test Result</th>
    </tr>
    <tr>
        <td colspan="2">merge-target-branch</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6175>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">build-docker-image</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6175>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">compile-nydus</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6175>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">compile-ctr-remote</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6175>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">compile-nydus-snapshotter</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6175>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">start-nydus-snapshotter-config-containerd</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6175>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">run-container-with-nydus-image</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6175>:white_check_mark:SUCCESS</a></td>
    </tr>
</table>

Sorry, the current test job result is not valid, because your pull request has been updated, and a new test job will be submitted.

**Comment by yqleng1987** [comment]
@yqleng1987 , the test job has been submitted. Please wait in patience.

**Comment by yqleng1987** [comment]
@yqleng1987 , The CI test is completed, please check result:
<table>
    <tr>
        <th colspan="2">Test Case</th><th>Test Result</th>
    </tr>
    <tr>
        <td colspan="2">merge-target-branch</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6220>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">build-docker-image</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6220>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">compile-nydus</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6220>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">compile-ctr-remote</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6220>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">compile-nydus-snapshotter</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6220>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">start-nydus-snapshotter-config-containerd</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6220>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">run-container-with-nydus-image</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/6220>:white_check_mark:SUCCESS</a></td>
    </tr>
</table>

Congratulations, your test job passed!

**Pull request merged**